plugins/mobile_channel_header_plug/mobile_channel_header_plug.test.jsx

```diff
@@ -137,13 +149,15 @@ describe('plugins/MobileChannelHeaderPlug', () => {
             dropdownText: 'some dropdown text',
         };
 
-        const wrapper = mount(
+        const wrapper = mountWithIntl(
             <MobileChannelHeaderPlug
                 components={[newTestPlug]}
                 channel={channel}
                 channelMember={channelMember}
                 theme={{}}
                 isDropdown={true}
+                appsEnabled={false}
+                actions={{doAppCall: jest.fn()}}
```

Comment:
Can we add one test where `appsEnabled` is true? Just to make sure `doAppCall` is called on click